src/modifiers/identity.py

```diff
@@ -0,0 +1,14 @@
+from talon import Module
+
+mod = Module()
+
+
+mod.list(
+    "cursorless_identity",
+    desc="Cursorless modifier for identity. Use to break inference.",
+)
+
+
+@mod.capture(rule="{user.cursorless_identity}")
+def cursorless_head_tail(m) -> dict:
```

Comment:
Minor copy-paste error, but I'm struggling to understand how the code even runs with this here? Ie have you tested "just"?

```suggestion
def cursorless_identity(m) -> dict:
```

Reply:
Tested and worked fine :p

Reply:
you don't find that surprising?  maybe it broke "head" and "tail"?  either way this change should fix it